code/modules/food_and_drinks/recipes/tablecraft/recipes_bread.dm

```diff
@@ -107,7 +107,7 @@
 	name = "Butterdog"
 	reqs = list(
 		/obj/item/food/bun = 1,
-		/obj/item/food/butter = 3,
+		/obj/item/food/butterslice = 1,
```

Comment:
If you look at the Butterdog sprite it looks like a one full butter is used. Maybe increase nutriments for Butterdog instead if you think that isn't enough

Reply:
I've made it take one stick of butter